cmd/fyne/commands/get.go

```diff
@@ -26,58 +40,33 @@ func NewGetter() *Getter {
 	return &Getter{}
 }
 
-// Get automates the download and install of a named GUI app package.
-func (g *Getter) Get(pkg string) error {
```

Comment:
This is part of a public API and cannot be removed - see usage at https://github.com/fyne-io/apps/blob/master/welcome.go#L164

Reply:
I was pretty sure that I didn't remove this. Will have to see what's up with that.

Reply:
It isn't removed. Look further down in the file.

Reply:
Oh I see. Can you keep the methods in the correct order so things don't get mixed up please?
Also the new static functions should appear after the type functions IIRC.